tests/user_examples/extra_src/shared/general_continuum/riemann_solver_extra.cpp

```diff
@@ -0,0 +1,13 @@
+#include "riemann_solver_extra.h"
+
+namespace SPH
+{
+	Vecd AcousticRiemannSolverExtra::DissipativePJumpExtra(const Vecd& u_jump, const Vecd& e_ij)
+	{
+		return rho0c0_geo_ave_ * u_jump * SMIN(Real(3) * SMAX(u_jump.dot(e_ij) * inv_c_ave_, Real(0)), Real(1));
+	}
+	Vecd DissipativeRiemannSolverExtra::DissipativePJumpExtra(const Vecd& u_jump, const Vecd& e_ij)
+	{
+		return rho0c0_geo_ave_ * u_jump;
+	}
+} // namespace SPH
```

Comment:
the term "geo" in "rho0c0_geo_ave_" is confusing, does it mean that this is a special term for geo material?

Reply:
The variable `"rho0c0_geo_ave_"` is defined in the original Riemann solver, I just inherit it in the `"riemann_solver_extra"`.